services/mac_manager/pom.xml

```diff
@@ -32,58 +32,28 @@
     </repositories>
 
     <dependencies>
+
+        <!--Service Basic dependencies-->
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-data-redis</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-thymeleaf</artifactId>
-        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-        <dependency>
-            <groupId>com.google.protobuf</groupId>
-            <artifactId>protobuf-java</artifactId>
-            <version>3.8.0</version>
-        </dependency>
-        <dependency>
-            <groupId>org.projectlombok</groupId>
-            <artifactId>lombok</artifactId>
-            <version>1.18.0</version>
-        </dependency>
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-pool2</artifactId>
             <version>2.4.2</version>
         </dependency>
-        <dependency>
-            <groupId>com.googlecode.json-simple</groupId>
-            <artifactId>json-simple</artifactId>
-            <version>1.1.1</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.ignite</groupId>
-            <artifactId>ignite-core</artifactId>
-            <version>2.8.0</version>
-        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
-            <exclusions>
```

Comment:
@kimeunju108 Could you review this part to see which lib is needed. Currently the mac_manager pom change caused UT failure: https://jenkins.alkaidcloud.io/job/alcor-controller-pr-test/372/console

Reply:
If we add org.junit.vintage back, MacManager basically won't run any UTs.

Reply:
@kimeunju108, as this PR include other important payload, I will temporarily disable UTs for mac manager and go ahead to merge the PR. 

Created an issue for tracking: https://github.com/futurewei-cloud/alcor/issues/199

Please fix the UTs with priority. A service can't sustain for long without active UT running.